events_test/src/main/java/com/adobe/aio/event/management/RegistrationServiceTester.java

```diff
@@ -41,6 +41,10 @@ public RegistrationServiceTester() {
         .build();
   }
 
+  public RegistrationService getRegistrationService(){
+    return this.registrationService;
+  }
```

Comment:
where is this used?

Reply:
it's use in further tests we do internally within Adobe ;-)